components/kyma-metrics-collector/pkg/process/public_cloud_info_test.go

```diff
@@ -118,17 +121,10 @@ func TestGetFeature(t *testing.T) {
 				Memory:   32,
 			},
 		},
-		{
-			cloudProvider: "aws",
-			vmType:        "t4g.nano",
```

Comment:
this type is no longer needed?

Reply:
yes, its not there in the main `resources/kcp/charts/kyma-metrics-collector/templates/public-cloud-info-config-map.yaml`